metron-interface/metron-rest/src/main/java/org/apache/metron/rest/controller/RestExceptionHandler.java

```diff
@@ -45,4 +45,14 @@ private HttpStatus getStatus(HttpServletRequest request) {
     }
     return HttpStatus.valueOf(statusCode);
   }
+
+  private String getFullMessage(Throwable ex) {
+    String fullMessage = ex.getMessage();
+    Throwable cause = ex.getCause();
```

Comment:
This is a possible recursion point, where  ex.getCause() == ex .
Can we use the commons ExceptionUtils.getRootCause()?  If not we need to guard against this.

Reply:
Yeah, I think this method is just `ExceptionUtils.getRootCause.getMessage()`, isn't it?

Reply:
I think so.  The get cause recursion is an oldie but a goodie.

Reply:
This is a blocker imo.